### DISCUSSION TITLE
Cruiser 3.0.36

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#370] [Announcements]
Cruiser 3.0.36 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):

- Save your routes in favorites
https://github.com/devemux86/cruiser/discussions/35

- Save your tracks in favorites
https://github.com/devemux86/cruiser/discussions/81

- BRouter (offline): configure the strength of avoidances
https://github.com/devemux86/cruiser/discussions/134

- Basic and extended settings
https://github.com/devemux86/cruiser/discussions/344

<img src="https://github.com/devemux86/cruiser/assets/3484020/95152bf5-dac6-492d-a99e-4feae8d3c5ab" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/d0af3f6a-548e-4928-b181-5205730b47ab" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c341cb60-90df-4547-963e-ce355bfb0844" width="33%"/>

**Comment by Leeb64** [comment]
It is becoming increasingly difficult to criticize something...
But I have something:
There is the "switch":Advanced settings, and a little further down under Advanced, the "switch":Advanced map menu.
You could actually save yourself the second switch and activate the expanded map menu with the first switch. That would be less confusing...

**Reply by devemux86** [reply]
We could do this, however they satisfy different needs:
- Many users after learning the basics of Cruiser can explore the extended settings to further customize the app

- Not many users would need the extended menu, as map / theme downloads would be enough for most cases

**Reply by Leeb64** [reply]
If someone doesn't know that this function exists, they probably won't find it  easyly

**Comment by Ciutnik** [comment]
Thank you very much for the great update! 
Chapeau bas! 👍

**Comment by DolomitesTom** [comment]
Yes thank you, great developments. May I ask what list entry is statistics per trip?

**Reply by devemux86** [reply]
@DolomitesTom we'll see, better not to make promises on development. :slightly_smiling_face: 

Focus of last release may have been the favorite routes / tracks,
but by the end, many other features were also implemented.

**Comment by devemux86** [comment]
Cruiser 3.0.35 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):
- https://github.com/devemux86/cruiser/discussions/375

**Comment by devemux86** [comment]
Cruiser 3.0.36 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):
- Improvements in search
